src/Elliptic/Systems/Xcts/FirstOrderSystem.hpp

```diff
@@ -0,0 +1,261 @@
+// Distributed under the MIT License.
+// See LICENSE.txt for details.
+
+#pragma once
+
+#include <cstddef>
+
+#include "DataStructures/DataBox/Prefixes.hpp"
+#include "DataStructures/Tensor/EagerMath/Magnitude.hpp"
+#include "Elliptic/Systems/Xcts/FluxesAndSources.hpp"
+#include "Elliptic/Systems/Xcts/Geometry.hpp"
+#include "Elliptic/Systems/Xcts/Tags.hpp"
+#include "NumericalAlgorithms/LinearOperators/PartialDerivatives.hpp"
+#include "PointwiseFunctions/GeneralRelativity/Tags.hpp"
+#include "Utilities/TMPL.hpp"
+
+namespace Xcts {
+
+/*!
+ * \brief The Extended Conformal Thin Sandwich (XCTS) decomposition of the
+ * Einstein constraint equations, formulated as a set of coupled first-order
+ * partial differential equations
+ *
+ * See \ref Xcts for details on the XCTS equations. This system introduces as
+ * auxiliary variables the conformal factor gradient \f$v_i=\partial_i\psi\f$,
+ * the gradient of the lapse times the conformal factor
+ * \f$w_i=\partial_i\left(\alpha\psi\right)\f$, and the symmetric shift strain
+ * \f$B_{ij}=\bar{D}_{(i}\beta_{j)}\f$. Note that \f$B_{ij}\f$ is the
+ * symmetrized covariant gradient of the shift vector field and analogous to the
+ * "strain" in an elasticity equation (see `Elasticity::FirstOrderSystem` and
+ * `Xcts::Tags::ShiftStrain` for details). From the strain we can compute the
+ * longitudinal operator by essentially removing its trace (see
+ * `Xcts::longitudinal_operator`).
+ *
+ * When we cast the equations in first-order flux-form
+ *
+ * \f[
+ * -\partial_i F^i_A + S_A = f_A(x)
+ * \f]
+ *
+ * (see also `Poisson::FirstOrderSystem`), the fluxes \f$F^i_A\f$, sources
+ * \f$S_A\f$ and fixed-sources \f$f_A\f$ are:
+ *
+ * \f{align}
+ * F^i_\psi &= \bar{\gamma}^{ij} v_j \\
+ * S_\psi &= -\bar{Gamma}^i_{ij}\bar{\gamma}^{jk}v_k
+ * + \frac{1}{8}\psi\bar{R} + \frac{1}{12}\psi^5 K^2
+ * - \frac{1}{8}\psi^{-7}\bar{A}^2 - 2\pi\psi^5\rho \\
+ * f_\psi &= 0 \\
+ * F^i_{\beta^j} &= \left(\bar{L}\beta\right)^{ij} \\
+ * S_{\beta^i} &= -\bar{\Gamma}^j_{jk} \left(\bar{L}\beta\right)^{ik}
```

Comment:
considering breaking this up over two lines

Reply:
👍 reformatted these equations